warehouse/templates/manage/projects.html

```diff
@@ -82,6 +83,15 @@ <h3 class="package-snippet__title">{{ project.name }}
           {% endif %}
         </div>
         <div class="package-snippet__buttons">
+
+          {% if not user_has_two_factor and project.name in projects_requiring_2fa %}
+          <a
+            href="{{ request.route_path('manage.account.two-factor') }}"
+            class="button button--danger"
+            title="{% trans %}This project requires 2FA to be enabled to manage{% endtrans %}"
```

Comment:
Since there are multiple reasons 2FA might be required (owners_require_2fa/pypi_mandates_2fa), it's probably best to surface which source is requiring this here.